contracts/utilities/TokenFeeCollector.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "../interfaces/IERC20.sol";
+import "../lib/Ownable.sol";
+import "../lib/SafeMath.sol";
+
+contract TokenFeeCollector is Ownable {
+    using SafeMath for uint;
+
+    // this means 100% so 1000000 is 1%, 100000 is 0.1%, etc
+    uint private constant maxBips = 100000000;
+
+    struct TokenInfo {
+        bool isValue;
+        address[] payees;
```

Comment:
Why not have this nested mapping?

i.e. mapping(address => mapping(address => Info))